concent_api/core/message_handlers.py

```diff
@@ -389,13 +389,14 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         task_to_compute,
     )
 
-    if Subtask.objects.filter(  # pylint: disable=no-member
-        subtask_id = task_to_compute.compute_task_def['subtask_id'],
-        state      = Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name,  # pylint: disable=no-member
-    ).exists():
-        return message.concents.ServiceRefused(
-            reason = message.concents.ServiceRefused.REASON.DuplicateRequest,
-        )
+    try:
```

Comment:
why `try ... except` instead of filtering?

Reply:
It is to reduce queries to database. I only one ask database for subtask, and then I check it's state- but I check local variable, not database. This is not perfect solution, it will be changed in https://github.com/golemfactory/concent/pull/778 anyway.

Reply:
actually here the number of requests to DB has not been changed

Reply:
maybe.. this will be resolved in https://github.com/golemfactory/concent/pull/778. In this place it is unified to be as it is in `handle_send_subtask_results_verify`